src/ophys_etl/modules/decrosstalk/__main__.py

```diff
@@ -114,7 +151,25 @@ def run(self):
             output_pair['planes'] = plane_pair
             coupled_planes.append(output_pair)
 
+            # purge movie data so that we are not carrying it around
+            # until the end of processing
+            plane_A.movie.purge_movie()
+            plane_B.movie.purge_movie()
+
         final_output['coupled_planes'] = coupled_planes
+
+        session_id = self.args['ophys_session_id']
+        figure_path = os.path.join(cache_dir,
+                                   f'{session_id}_roi_fig.png')
+
+        generate_roi_figure(session_id,
+                            planes_for_plotting,
+                            final_output,
+                            figure_path)
```

Comment:
for `figure_path` and `qc_fname` above, should these be added to the output_schema, and the LIMS strategy updated?
Is there any intent to track these files.
Without this, how will we know which QC files are associated with a particular result, besides just infering based on NFS location?

Reply:
According to Wayne (from an email exchange we had)

> I am going to stop short of "making them available through LIMS".  I think it is sufficient if there are some consistently named files that live in the 'decrosstalk' directory.

I think, for now, the onus is on whoever wants to use these files to find them based on the decrosstalk input.json, which will point to the `qc_output_dir`, which is the parent of all of these file locations.

Reply:
ok